newIDE/app/src/MainFrame/index.js

```diff
@@ -1130,9 +1179,11 @@ class MainFrame extends React.Component<Props, State> {
     this.setState(
       {
         editorTabs: openEditorTab(this.state.editorTabs, {
-          name: name + ' ' + i18n._(t`(Extension)`),
+          label: name + ' ' + i18n._(t`(Extension)`),
           renderEditor: ({ isActive, editorRef }) => (
             <EventsFunctionsExtensionEditor
+              onPreview={null}
+              previewButtonSettings={emptyPreviewButtonSettings}
```

Comment:
Do you need previewButtonSettings, because EventsFunctionsExtensionEditor itself is passing emptyPreviewButtonSettings.
Same, do you need onPreview?

If not let's remove them - the less the props, the clearer it is.